src/fswAlgorithms/sensorInterfaces/scanningInstrumentController/scanningInstrumentController.c

```diff
@@ -0,0 +1,107 @@
+/*
+ ISC License
+
+ Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado Boulder
+
+ Permission to use, copy, modify, and/or distribute this software for any
+ purpose with or without fee is hereby granted, provided that the above
+ copyright notice and this permission notice appear in all copies.
+
+ THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+*/
+
+
+#include "fswAlgorithms/sensorInterfaces/scanningInstrumentController/scanningInstrumentController.h"
+#include "architecture/utilities/linearAlgebra.h"
+#include "string.h"
+
+/*!
+    This method initializes the output messages for this module.
+ @return void
+ @param configData The configuration data associated with this module
+ @param moduleID The module identifier
+ */
+void SelfInit_scanningInstrumentController(scanningInstrumentControllerConfig  *configData, int64_t moduleID)
+{
+    DeviceCmdMsg_C_init(&configData->deviceCmdOutMsg);
+}
+
```

Comment:
:large_blue_circle: Extra new line can be removed.

Reply:
Extra line removed.